interactive_engine/compiler/src/test/java/com/alibaba/graphscope/cypher/antlr4/MatchTest.java

```diff
@@ -122,4 +123,15 @@ public void match_6_test() {
         Assert.assertEquals(
                 SqlTypeName.BIGINT, condition.getOperands().get(1).getType().getSqlTypeName());
     }
+
+    @Test
+    public void match_7_test() {
```

Comment:
Should we include two more cases, one with no parameter, and one with more than one parameters of different types?

Reply:
fix in https://github.com/alibaba/GraphScope/pull/3140/commits/391dc7b4d4f95aa5247c773f21ed1fa04cc370dc